civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -20,8 +20,12 @@
       overflow: visible !important;
     }
 
-    #bootstrap-theme .dropdown-menu[datepicker-popup-wrap] {
-      width: 316px;
+    #bootstrap-theme {
+      display: inline-block;
+
+      [uib-datepicker-popup-wrap] .dropdown-menu {
+        width: 316px;
```

Comment:
To set the calendar width. https://github.com/compucorp/civihr-employee-portal-theme/pull/62#discussion_r64066874